Resources/Prototypes/_Goobstation/Entities/Mobs/Species/yowie.yml

```
@@ -0,0 +1,55 @@
+    species: Yowie
+    hideLayersOnEquip:
+    - Hair
+    - Snout
+  - type: Temperature # Fur = bad with heat, stole numbers from moth
+    heatDamageThreshold: 320
+    coldDamageThreshold: 230
+    currentTemperature: 310.15
+    specificHeat: 46
+    coldDamage:
+      types:
+        Cold : 0.05 #per second, scales with temperature & other constants
```

yamllint

[warning] too many spaces before colon

(colons)